lib/matplotlib/streamplot.py

```diff
@@ -310,19 +310,19 @@ class Grid:
     """Grid of data."""
     def __init__(self, x, y):
 
-        if x.ndim == 1:
+        if np.ndim(x) == 1:
             pass
-        elif x.ndim == 2:
+        elif np.ndim(x) == 2:
             x_row = x[0, :]
```

Comment:
If `x` is not an array `x[0, :]` will likely fail.

Reply:
indeed, fixed.